vault/dynamic_system_view.go

```diff
@@ -274,9 +274,9 @@ func (d dynamicSystemView) ListVersionedPlugins(ctx context.Context, pluginType
 	return d.core.pluginCatalog.ListVersionedPlugins(ctx, pluginType)
 }
 
-// MlockEnabled returns the configuration setting for enabling mlock on plugins.
+// OpenBao no longer uses mlock but MlockEnabled is retained for plugin compatibility.
```

Comment:
I think this is the right approach; I don't see any callers of this function but since it is defined in the interfaces, we can't remove it.

Reply:
Yes, I tried ripping it out and OpenBao then failed to compile because `dynamicSystemView` wasn't implementing a part of the `pluginutil.RunnerUtil` interface.